devicetypes/smartthings/zigbee-battery-accessory-dimmer.src/zigbee-battery-accessory-dimmer.groovy

```diff
@@ -118,9 +126,50 @@ def handleSengledSwitchEvents(descMap) {
 	}
 }
 
+def handleCentraliteSmartSwitchEvents(descMap) {
+	if (descMap.clusterInt == 0x0006) {
```

Comment:
use `zigbee.ONOFF_CLUSTER` and `zigbee.LEVEL_CONTROL_CLUSTER` instead of 0x0006 and 0x0008

Reply:
I believe that this event can be taken care of with code that exists in the Zigbee library already:

```
def event = zigbee.getEvent(description)
	if (event) {
		results << createEvent(event)
	}  else {
// the rest of parse
	}
```

Reply:
Ok

Reply:
It occurred to me last night as I was falling asleep that the zigbee.getEvent code won't handle that because the data is coming in as a command and not an attribute. So yeah, no need to worry about the zigbee.getEvent stuff.